setup.py

```diff
@@ -94,11 +94,13 @@
     'invenio-search-ui>=1.0.0a2',
     'invenio-search>=1.0.0a9',
     'invenio-theme==1.0.0b2',
+    'invenio-xrootd>=1.0.0a4',
     'mistune>=0.7.4',
     'py-gfm>=0.1.3',
     'pymdown-extensions>=3.5',
     'python-markdown-math>=0.3',
     'python-slugify>=1.2.4',
+    'raven>=6.3.0',  # required by invenio-logging
```

Comment:
Shall we add it to `requirements-production.txt` already?

Reply:
Done